packages/botonic-react/src/webchat/message-list/scroll-button.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useContext, useEffect, useState } from 'react'
+
+import ArrawScrollDown from '../../assets/arrow-scroll-down.svg'
```

Comment:
typo `ArrowScrollDown`

Reply:
I don't see it the same way. The MessagesList component I think should not have the logic of whether the UnreadMessageBanner component is activated or if it is a custom (since you can activate it both ways), this is part of the logic of the UnreadMessageBanner.

Reply:
Applied